packages/adapter-node/README.md

```diff
@@ -60,77 +56,33 @@ env: {
 MY_HOST_VARIABLE=127.0.0.1 MY_PORT_VARIABLE=4000 node build
 ```
 
-## Middleware
+## Custom server
 
-The adapter exports a middleware `(req, res, next) => {}` that's compatible with [Express](https://github.com/expressjs/expressjs.com) / [Connect](https://github.com/senchalabs/connect) / [Polka](https://github.com/lukeed/polka). Additionally, it also exports a reference server implementation using this middleware with a plain Node HTTP server.
+The adapter creates two files in your build directory — `index.js` and `handler.js`. Running `index.js` — e.g. `node build`, if you use the default build directory — will start a server on the configured port.
 
-But you can use your favorite server framework to combine it with other middleware and server logic. You can import `kitMiddleware`, your ready-to-use SvelteKit middleware from the `build` directory. You can use [the `entryPoint` option](#entryPoint) to bundle your custom server entry point.
+Alternatively, you can import the `handler.js` file, which exports a handler suitable for use with [Express](https://github.com/expressjs/expressjs.com), [Connect](https://github.com/senchalabs/connect) or [Polka](https://github.com/lukeed/polka) (or even just the built-in [`http.createServer`](https://nodejs.org/dist/latest/docs/api/http.html#httpcreateserveroptions-requestlistener)) and set up your own server:
 
 ```js
-// src/server.js
-import { assetsMiddleware, prerenderedMiddleware, kitMiddleware } from '../build/middlewares.js';
-import polka from 'polka';
-
-const app = polka();
+// my-server.js
+import { handler } from './build/handler.js';
+import express from 'express';
 
-const myMiddleware = function (req, res, next) {
-	console.log('Hello world!');
-	next();
-};
-
-app.use(myMiddleware);
+const app = express();
 
-app.get('/no-svelte', (req, res) => {
-	res.end('This is not Svelte!');
+// add a route that lives separately from the SvelteKit app
+app.get('/healthcheck', (req, res) => {
+	res.end('ok');
 });
 
-app.all('*', assetsMiddleware, prerenderedMiddleware, kitMiddleware);
-
-// Express users can also write in a second way:
-// app.use(assetsMiddleware, prerenderedMiddleware, kitMiddleware);
-
-app.listen(3000);
-```
-
-For using middleware in dev mode, [see the FAQ](https://kit.svelte.dev/faq#how-do-i-use-x-with-sveltekit-how-do-i-use-middleware).
-
-## Advanced Configuration
-
-### esbuild
+// let SvelteKit handle everything else, including serving
+// prerendered pages and static assets
```

Comment:
this seems like a short enough comment it could it on one line. or at least I'd move the break later in the sentence
```suggestion
// let SvelteKit handle everything else, including serving prerendered pages and static assets
```